### PR TITLE
updated board references from BRD4161A to BRD4187C in example commands

### PR DESCRIPTION
**Pull request by wehale**
Silicon Labs is recommending the use of the EFR32MG24, as such I am updating our board references in our example documentation to use the BRD4187C as a default instead of BRD4161A.

### CODE REVIEW

File: examples/light-switch-app/silabs/efr32/README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Silicon Labs platform.
           $ cd ~/connectedhomeip/examples/light-switch-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export EFR32_BOARD=BRD4161A
+          $ export EFR32_BOARD=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/light-switch-app/silabs/efr32/README.md:121]: While you are in those files `EFR32_BOARD` has been renamed to `SILABS_BOARD`. Can you update that too
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

File: examples/light-switch-app/silabs/efr32/README.md [hunk 2]
```diff
@@ -129,26 +129,26 @@ Silicon Labs platform.
...
-          $ ./scripts/examples/gn_efr32_example.sh examples/light-switch-app/silabs/efr32/ out/light-switch-app_rpc BRD4161A 'import("//with_pw_rpc.gni")'
+          $ ./scripts/examples/gn_efr32_example.sh examples/light-switch-app/silabs/efr32/ out/light-switch-app_rpc BRD4187C 'import("//with_pw_rpc.gni")'
 
     or use GN/Ninja Directly
 
           $ cd ~/connectedhomeip/examples/light-switch-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export EFR32_BOARD=BRD4161A
+          $ export EFR32_BOARD=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/light-switch-app/silabs/efr32/README.md:151]:
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

File: examples/lighting-app/silabs/efr32/README.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ Silicon Labs platform.
           $ cd ~/connectedhomeip/examples/lighting-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export silabs_board=BRD4161A
+          $ export silabs_board=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/lighting-app/silabs/efr32/README.md:115]:
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

File: examples/lighting-app/silabs/efr32/README.md [hunk 4]
```diff
@@ -123,22 +123,22 @@ Silicon Labs platform.
...
-          $ ./scripts/examples/gn_efr32_example.sh examples/lighting-app/silabs/efr32/ out/lighting_app_rpc BRD4161A 'import("//with_pw_rpc.gni")'
+          $ ./scripts/examples/gn_efr32_example.sh examples/lighting-app/silabs/efr32/ out/lighting_app_rpc BRD4187C 'import("//with_pw_rpc.gni")'
 
     or use GN/Ninja Directly
 
           $ cd ~/connectedhomeip/examples/lighting-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export silabs_board=BRD4161A
+          $ export silabs_board=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/lighting-app/silabs/efr32/README.md:141]:
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

File: examples/lock-app/silabs/efr32/README.md [hunk 5]
```diff
@@ -118,7 +118,7 @@ Mac OS X
           $ cd ~/connectedhomeip/examples/silabs/lock-app/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export EFR32_BOARD=BRD4161A
+          $ export EFR32_BOARD=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/lock-app/silabs/efr32/README.md:121]:
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

File: examples/lock-app/silabs/efr32/README.md [hunk 6]
```diff
@@ -154,7 +154,7 @@ Mac OS X
           $ cd ~/connectedhomeip/examples/lock-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export EFR32_BOARD=BRD4161A
+          $ export EFR32_BOARD=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/lock-app/silabs/efr32/README.md:157]:
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

File: examples/window-app/silabs/efr32/README.md [hunk 7]
```diff
@@ -111,7 +111,7 @@ Silicon Labs platform.
           $ cd ~/connectedhomeip/examples/window-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export EFR32_BOARD=BRD4161A
+          $ export EFR32_BOARD=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/window-app/silabs/efr32/README.md:114]:
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

File: examples/window-app/silabs/efr32/README.md [hunk 8]
```diff
@@ -122,22 +122,22 @@ Silicon Labs platform.
...
-          $ ./scripts/examples/gn_efr32_example.sh examples/window-app/silabs/efr32/ out/window_app_rpc BRD4161A 'import("//with_pw_rpc.gni")'
+          $ ./scripts/examples/gn_efr32_example.sh examples/window-app/silabs/efr32/ out/window_app_rpc BRD4187C 'import("//with_pw_rpc.gni")'
 
     or use GN/Ninja Directly
 
           $ cd ~/connectedhomeip/examples/window-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export EFR32_BOARD=BRD4161A
+          $ export EFR32_BOARD=BRD4187C
```
Comment (jmartinez-silabs) [inline examples/window-app/silabs/efr32/README.md:140]:
```suggestion
          $ export SILABS_BOARD=BRD4187C
```

**Review by jmartinez-silabs — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**